### PR TITLE
adding new package details

### PR DESCRIPTION
**Pull request by sdwrage**
Added in some package details like name, description, license, etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "test",
+  "name": "Phalanx",
```
Comment (sdwrage) [inline package.json:2]: Thought that it would be better than test :P

File: package.json [hunk 2]
```diff
@@ -30,8 +30,8 @@
   "main": "app.js",
   "repository": {
     "type": "git",
-    "url": "git://github.com/vagrant/test.git"
+    "url": "git://github.com/spartaneous/phalanx.git"
   },
-  "author": "vagrant",
-  "license": ""
+  "author": "Spartaneous",
+  "license": "MIT"
```
Comment (sdwrage) [inline package.json:36]: Is this right?

File: package.json [hunk 3]
```diff
@@ -30,8 +30,8 @@
   "main": "app.js",
   "repository": {
     "type": "git",
-    "url": "git://github.com/vagrant/test.git"
+    "url": "git://github.com/spartaneous/phalanx.git"
```
Comment (sdwrage) [inline package.json:33]: Didn't know if this was my fork or the main repo so I just went ahead and said main fork as that seemed about right.

**Comment by ddelrio1986** [comment]
This all looks good.

**Pull request merged**